src/plugins/intel_gpu/src/plugin/transformations_pipeline.cpp

```diff
@@ -421,10 +407,6 @@ void TransformationsPipeline::apply(std::shared_ptr<ov::Model> func) {
         pass_config->disable<ngraph::pass::ConvertSoftMax8ToSoftMax1>();
         pass_config->enable<ngraph::pass::ConvertGather8ToGather7>();
 
-        if (!config.enable_loop_unrolling) {
```

Comment:
looks like this change can break logic for our customers
we have to ConvertTensorIteratorToSequence if enable_loop_unrolling is true
@paularamo

Reply:
https://github.com/openvinotoolkit/openvino/pull/14084